src/components/org/apache/jmeter/control/IncludeController.java

```diff
@@ -121,19 +121,20 @@ protected HashTree loadIncludedElements() {
         final String includePath = getIncludePath();
         HashTree tree = null;
         if (includePath != null && includePath.length() > 0) {
+            String fileName=prefix+includePath;
             try {
-                String fileName=prefix+includePath;
+                
                 File file = new File(fileName);
                 final String absolutePath = file.getAbsolutePath();
                 log.info("loadIncludedElements -- try to load included module: "+absolutePath);
                 if(!file.exists() && !file.isAbsolute()){
                     log.info("loadIncludedElements -failed for: "+absolutePath);
                     file = new File(FileServer.getFileServer().getBaseDir(), includePath);
                     log.info("loadIncludedElements -Attempting to read it from: " + file.getAbsolutePath());
-                    if(!file.exists()){
+                    if(!file.canRead() || !file.isFile()){
```

Comment:
Hi,
Why do you replace !file.exists() by !file.canRead() || !file.isFile() 
Thanks